dlpy/model.py

```diff
@@ -232,10 +232,10 @@ def fit(self, data, inputs=None, target=None, data_specs=None, mini_batch_size=1
             of the labels or order them in the order they are recieved with
             training data samples.
             Valid Values: 'ascending', 'descending', 'hash'
-            Default: 'ascending'
+            Default : 'ascending'
```

Comment:
This space should be removed again.  Spaces after quotes are only used in the data type definition.